tests/conftest/kubernetes.rego

```diff
@@ -0,0 +1,17 @@
+package kubernetes
```

Comment:
As already commented on `defaults.rego`, I would drop this. That is, unless functions (or what ever Rego names them) can not be defined in `defaults.rego`.

But then either use everything defined in here or drop what is not used.